netctl/netctl.go

```diff
@@ -237,23 +235,18 @@ func listNetworks(ctx *cli.Context) {
 	} else {
 		writer := tabwriter.NewWriter(os.Stdout, 0, 2, 2, ' ', 0)
 		defer writer.Flush()
-		writer.Write([]byte("Tenant\tNetwork\tPublic\tEncap\tSubnet\tGateway\n"))
-		writer.Write([]byte("------\t-------\t------\t-----\t------\t-------\n"))
+		writer.Write([]byte("Tenant\tNetwork\tEncap\tTag\tSubnet\tGateway\n"))
```

Comment:
@mapuri our comments crossed. I'm ok with "Encap Type" and "Packet Tag" also. encap-type encamp-value might be too geeky :)